packages/components/src/number-control/test/index.tsx

```diff
@@ -0,0 +1,616 @@
+/**
+ * External dependencies
+ */
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import NumberControl from '..';
+import type { NumberControlProps } from '../types';
+
+const getInput = () => screen.getByRole( 'spinbutton' ) as HTMLInputElement;
```

Comment:
I think we could get rid of that helper function and inline those calls.

Reply:
Sure thing!